.gitignore

```diff
@@ -7,6 +7,6 @@ npm-debug.log
 package-lock.json
 /en-us/
 /zh-cn/
-index.html
+/index.html
```

Comment:
Why change here ?

Reply:
saga_designer/index.html must not be ignored.